admin/controllers/pupil-pin.js

```diff
@@ -122,14 +122,21 @@ const getGeneratedPinsList = async (req, res, next) => {
   })
 }
 
+/**
+ * Get Print PINs.
+ * @param req
+ * @param res
+ * @param next
+ * @returns {Promise<*>}
+ */
 const getPrintPins = async (req, res, next) => {
   res.locals.pageTitle = 'Print pupils'
   let pupils
   let school
   let qrDataURL
   const date = dateService.formatDayAndDate(new Date())
   try {
-    pupils = await pinService.getPupilsWithActivePins(req.user.School)
+    pupils = await pinService.getPupilsWithActivePins(req.user.School, req.user.schoolId)
```

Comment:
why do you need 2 identifiers for the school?

Reply:
@GuyHarwood because ```getPupilsWithActivePins``` will now also get groups using ```getGroupsAsArray``` and this method needs the schoolId, not the dfeNumber.